code/datums/supply_packs/supplies.dm

```diff
@@ -0,0 +1,502 @@
+//////SUPPLIES//////
+
+/datum/supply_packs/toner
+	name = "Toner cartridges"
+	contains = list(/obj/item/device/toner,
+					/obj/item/device/toner,
+					/obj/item/device/toner,
+					/obj/item/device/toner,
+					/obj/item/device/toner,
+					/obj/item/device/toner)
+	cost = 10
+	containertype = /obj/structure/closet/crate/basic
+	containername = "toner cartridges crate"
+	group = "Supplies"
+	containsdesc = "For when the clown decides to use up all of your scanner ink. Contains an excessive six sets of replacement toner."
```

Comment:
Scanners don't use ink, printers do.

Reply:
In that case, we'll change it to the all-in-one photocopier. Standard on Box's Cargo!